qt6: 6.4.3 -> 6.5.0

NickCao opened the pull request:
###### Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

NickCao committed:
qt6: 6.4.3 -> 6.5.0

NickCao committed:
qt6.qtlocation: init at 6.5.0

NickCao committed:
qt6.qtquickeffectmaker: init at 6.5.0

NickCao committed:
qt6.qtgrpc: init at 6.5.0

NickCao committed:
qt6.full: add missing modules

NickCao commented:
Result of `nixpkgs-review pr 224469` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages failed to build:</summary>
  <ul>
    <li>copyq</li>
    <li>jami</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>mediaelch-qt6</li>
    <li>obs-studio-plugins.obs-ndi</li>
    <li>texworks</li>
    <li>zeal-qt6</li>
  </ul>
</details>
<details>
  <summary>161 packages built:</summary>
  <ul>
    <li>adwaita-qt6</li>
    <li>adwaita-qt6.dev</li>
    <li>alice-tools-qt6</li>
    <li>bambootracker-qt6</li>
    <li>beamerpresenter</li>
    <li>beamerpresenter-poppler</li>
    <li>calcmysky</li>
    <li>calibre</li>
    <li>cudaPackages.nsight_compute</li>
    <li>cutemaze</li>
    <li>dolphin-emu</li>
    <li>duckstation</li>
    <li>fcitx5-chinese-addons</li>
    <li>fcitx5-configtool</li>
    <li>fcitx5-unikey</li>
    <li>fcitx5-with-addons</li>
    <li>gpxsee-qt6</li>
    <li>gqrx</li>
    <li>gqrx-gr-audio</li>
    <li>gqrx-portaudio</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>kime</li>
    <li>ladybird</li>
    <li>libsForQt5.fcitx5-qt</li>
    <li>maestral-gui</li>
    <li>maestral-gui.dist</li>
    <li>mozillavpn</li>
    <li>obs-studio</li>
    <li>obs-studio-plugins.advanced-scene-switcher</li>
    <li>obs-studio-plugins.droidcam-obs</li>
    <li>obs-studio-plugins.input-overlay</li>
    <li>obs-studio-plugins.looking-glass-obs</li>
    <li>obs-studio-plugins.obs-gstreamer</li>
    <li>obs-studio-plugins.obs-hyperion</li>
    <li>obs-studio-plugins.obs-livesplit-one</li>
    <li>obs-studio-plugins.obs-move-transition</li>
    <li>obs-studio-plugins.obs-multi-rtmp</li>
    <li>obs-studio-plugins.obs-nvfbc</li>
    <li>obs-studio-plugins.obs-pipewire-audio-capture</li>
    <li>obs-studio-plugins.obs-source-record</li>
    <li>obs-studio-plugins.obs-vaapi</li>
    <li>obs-studio-plugins.obs-vkcapture</li>
    <li>obs-studio-plugins.wlrobs</li>
    <li>olive-editor</li>
    <li>pgmodeler</li>
    <li>pokefinder</li>
    <li>prismlauncher</li>
    <li>punes-qt6</li>
    <li>python310Packages.pyqt6</li>
    <li>python310Packages.pyqt6-charts</li>
    <li>python310Packages.pyqt6-charts.dist</li>
    <li>python310Packages.pyqt6-webengine</li>
    <li>python310Packages.pyqt6-webengine.dev</li>
    <li>python310Packages.pyqt6-webengine.dist</li>
    <li>python310Packages.pyqt6.dev</li>
    <li>python310Packages.pyqt6.dist</li>
    <li>python311Packages.pyqt6</li>
    <li>python311Packages.pyqt6-charts</li>
    <li>python311Packages.pyqt6-charts.dist</li>
    <li>python311Packages.pyqt6-webengine</li>
    <li>python311Packages.pyqt6-webengine.dev</li>
    <li>python311Packages.pyqt6-webengine.dist</li>
    <li>python311Packages.pyqt6.dev</li>
    <li>python311Packages.pyqt6.dist</li>
    <li>qalculate-qt</li>
    <li>qgnomeplatform-qt6</li>
    <li>qmmp</li>
    <li>qt6.full</li>
    <li>qt6.qmake</li>
    <li>qt6.qt3d</li>
    <li>qt6.qt3d.dev</li>
    <li>qt6.qt5compat</li>
    <li>qt6.qt5compat.dev</li>
    <li>qt6.qtbase</li>
    <li>qt6.qtbase.dev</li>
    <li>qt6.qtcharts</li>
    <li>qt6.qtcharts.dev</li>
    <li>qt6.qtconnectivity</li>
    <li>qt6.qtconnectivity.dev</li>
    <li>qt6.qtdatavis3d</li>
    <li>qt6.qtdatavis3d.dev</li>
    <li>qt6.qtdeclarative</li>
    <li>qt6.qtdeclarative.dev</li>
    <li>qt6.qtdoc</li>
    <li>qt6.qtgrpc</li>
    <li>qt6.qtgrpc.dev</li>
    <li>qt6.qthttpserver</li>
    <li>qt6.qthttpserver.dev</li>
    <li>qt6.qtimageformats</li>
    <li>qt6.qtimageformats.dev</li>
    <li>qt6.qtlanguageserver</li>
    <li>qt6.qtlanguageserver.dev</li>
    <li>qt6.qtlocation</li>
    <li>qt6.qtlocation.dev</li>
    <li>qt6.qtlottie</li>
    <li>qt6.qtlottie.dev</li>
    <li>qt6.qtmultimedia</li>
    <li>qt6.qtmultimedia.dev</li>
    <li>qt6.qtnetworkauth</li>
    <li>qt6.qtnetworkauth.dev</li>
    <li>qt6.qtpositioning</li>
    <li>qt6.qtpositioning.dev</li>
    <li>qt6.qtquick3d</li>
    <li>qt6.qtquick3d.dev</li>
    <li>qt6.qtquick3dphysics</li>
    <li>qt6.qtquick3dphysics.dev</li>
    <li>qt6.qtquickeffectmaker</li>
    <li>qt6.qtquickeffectmaker.dev</li>
    <li>qt6.qtquicktimeline</li>
    <li>qt6.qtquicktimeline.dev</li>
    <li>qt6.qtremoteobjects</li>
    <li>qt6.qtremoteobjects.dev</li>
    <li>qt6.qtscxml</li>
    <li>qt6.qtscxml.dev</li>
    <li>qt6.qtsensors</li>
    <li>qt6.qtsensors.dev</li>
    <li>qt6.qtserialbus</li>
    <li>qt6.qtserialbus.dev</li>
    <li>qt6.qtserialport</li>
    <li>qt6.qtserialport.dev</li>
    <li>qt6.qtshadertools</li>
    <li>qt6.qtshadertools.dev</li>
    <li>qt6.qtspeech</li>
    <li>qt6.qtspeech.dev</li>
    <li>qt6.qtsvg</li>
    <li>qt6.qtsvg.dev</li>
    <li>qt6.qttools</li>
    <li>qt6.qttools.dev</li>
    <li>qt6.qttranslations</li>
    <li>qt6.qttranslations.dev</li>
    <li>qt6.qtvirtualkeyboard</li>
    <li>qt6.qtvirtualkeyboard.dev</li>
    <li>qt6.qtwayland</li>
    <li>qt6.qtwayland.dev</li>
    <li>qt6.qtwebchannel</li>
    <li>qt6.qtwebchannel.dev</li>
    <li>qt6.qtwebengine</li>
    <li>qt6.qtwebengine.dev</li>
    <li>qt6.qtwebsockets</li>
    <li>qt6.qtwebsockets.dev</li>
    <li>qt6.qtwebview</li>
    <li>qt6.qtwebview.dev</li>
    <li>qt6Packages.kcoreaddons</li>
    <li>qt6Packages.kcoreaddons.bin</li>
    <li>qt6Packages.kcoreaddons.dev</li>
    <li>qt6Packages.poppler</li>
    <li>qt6Packages.poppler.dev</li>
    <li>qt6Packages.qtpbfimageplugin</li>
    <li>qt6Packages.quazip</li>
    <li>qt6Packages.quazip.dev</li>
    <li>qt6Packages.qxlsx</li>
    <li>qtwirediff</li>
    <li>qutebrowser-qt6</li>
    <li>qutebrowser-qt6.dist</li>
    <li>retext</li>
    <li>retext.dist</li>
    <li>stellarium</li>
    <li>tdesktop</li>
    <li>zint</li>
    <li>zint.dev</li>
  </ul>
</details>

NickCao commented:
Mostly goes well but with a few incompatibilities:

- [ ] copyq
- [ ] jami
- [ ] khoj
- [ ] khoj.dist
- [ ] mediaelch-qt6
- [ ] texworks
- [ ] zeal-qt6

linsui commented:
Jami upstream issue https://git.jami.net/savoirfairelinux/jami-daemon/-/issues/841.

peterhoeg commented:
@GrahamcOfBorg build zeal-qt6

peterhoeg commented:
unless I'm misunderstanding things, qt6 6.5 now has a `qtqml` module which we will need for zeal but is not part of this PR.

NickCao commented:
> unless I'm misunderstanding things, qt6 6.5 now has a `qtqml` module which we will need for zeal but is not part of this PR.

At least am not seeing it on https://download.qt.io/official_releases/qt/6.5/6.5.0/submodules/. And the release note only mentions improvement on existing qml toolings not new modules?

peterhoeg commented:
This one, but it’s also part of 6.4: <https://doc.qt.io/qt-6/qtqml-index.html>

It could be that it’s part of another and that’s just what I’m missing to bring in.

NickCao commented:
>  It could be that it’s part of another and that’s just what I’m missing to bring in.

Part of `qtdeclarative`

Stunkymonkey commented:
`mediaelch-qt6` seems to me more an implementation problem not an packaging one. will report this upstream.

Artturin commented:
Copyq https://github.com/hluk/CopyQ/issues/2323

Artturin commented:
copyq upstream pr https://github.com/hluk/CopyQ/pull/2324

builds but i get

```
ERROR: Cannot load library /nix/store/13sxagadpsf3m26cn169gjd62rgabgnn-CopyQ-6.4.0/lib/copyq/plugins/libitemtags.so: (/nix/store/13sxagadpsf3m26cn169gjd62rgabgnn-CopyQ-6.4.0/lib/copyq/plugins/libitemtags.so: undefined symbol: _ZrsR11QDataStreamR7Command)
```

when launching but the program seems to work ok regardless.

Artturin commented:
copyq upstream pr https://github.com/hluk/CopyQ/pull/2324

builds but i get

`
ERROR: Cannot load library /nix/store/13sxagadpsf3m26cn169gjd62rgabgnn-CopyQ-6.4.0/lib/copyq/plugins/libitemtags.so: (/nix/store/13sxagadpsf3m26cn169gjd62rgabgnn-CopyQ-6.4.0/lib/copyq/plugins/libitemtags.so: undefined symbol: _ZrsR11QDataStreamR7Command)
`

when launching but the program seems to work ok regardless.

peterhoeg commented:
Can I trouble you to try and pull in 9a4d9ebd74a7b5754e470ca1eb97f9677e081b7b to see if that sorts out zeal-qt6?

NickCao commented:
> Can I trouble you to try and pull in [9a4d9eb](https://github.com/NixOS/nixpkgs/commit/9a4d9ebd74a7b5754e470ca1eb97f9677e081b7b) to see if that sorts out zeal-qt6?

Sadly still fails with
```
zeal> CMake Error at src/app/CMakeLists.txt:36 (target_link_libraries):
zeal>   Error evaluating generator expression:
zeal>     $<TARGET_PROPERTY:Qt6::QmlIntegration,_qt_module_has_private_headers>
zeal>   Target "Qt6::QmlIntegration" not found.
```
But the relevant cmake files are indeed in qtdeclarative.

peterhoeg commented:
For now though, there's not much point holding up this change as zeal-qt5 integrates much better anyway, so that's what people are more likely to be using. I'm completely OK with marking zeal-qt6 as broken.

NickCao committed:
qt6.qtbase: refresh patches

NickCao committed:
qt6: move mkspecs/modules to dev output

NickCao committed:
copyq: 7.0.0 -> unstable-2023-04-14

NickCao committed:
zeal-qt6: fix build with qt 6.5

NickCao commented:
The remaining blockers are now jami and texworks.

Result of `nixpkgs-review pr 224469` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages failed to build:</summary>
  <ul>
    <li>jami</li>
    <li>mediaelch-qt6</li>
    <li>obs-studio-plugins.obs-ndi</li>
    <li>texworks</li>
  </ul>
</details>
<details>
  <summary>176 packages built:</summary>
  <ul>
    <li>adwaita-qt6</li>
    <li>adwaita-qt6.dev</li>
    <li>alice-tools-qt6</li>
    <li>anki</li>
    <li>anki.dist</li>
    <li>anki.doc</li>
    <li>anki.man</li>
    <li>bambootracker-qt6</li>
    <li>beamerpresenter</li>
    <li>beamerpresenter-poppler</li>
    <li>calcmysky</li>
    <li>calibre</li>
    <li>copyq</li>
    <li>cudaPackages.nsight_compute</li>
    <li>cutemaze</li>
    <li>dolphin-emu</li>
    <li>duckstation</li>
    <li>fcitx5-chinese-addons</li>
    <li>fcitx5-configtool</li>
    <li>fcitx5-unikey</li>
    <li>fcitx5-with-addons</li>
    <li>gpxsee-qt6</li>
    <li>gqrx</li>
    <li>gqrx-gr-audio</li>
    <li>gqrx-portaudio</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>kime</li>
    <li>ladybird</li>
    <li>libsForQt5.fcitx5-qt</li>
    <li>maestral-gui</li>
    <li>maestral-gui.dist</li>
    <li>mnemosyne</li>
    <li>mnemosyne.dist</li>
    <li>mozillavpn</li>
    <li>nanovna-saver</li>
    <li>nanovna-saver.dist</li>
    <li>obs-studio</li>
    <li>obs-studio-plugins.advanced-scene-switcher</li>
    <li>obs-studio-plugins.droidcam-obs</li>
    <li>obs-studio-plugins.input-overlay</li>
    <li>obs-studio-plugins.looking-glass-obs</li>
    <li>obs-studio-plugins.obs-backgroundremoval</li>
    <li>obs-studio-plugins.obs-gstreamer</li>
    <li>obs-studio-plugins.obs-hyperion</li>
    <li>obs-studio-plugins.obs-livesplit-one</li>
    <li>obs-studio-plugins.obs-move-transition</li>
    <li>obs-studio-plugins.obs-multi-rtmp</li>
    <li>obs-studio-plugins.obs-nvfbc</li>
    <li>obs-studio-plugins.obs-pipewire-audio-capture</li>
    <li>obs-studio-plugins.obs-source-record</li>
    <li>obs-studio-plugins.obs-vaapi</li>
    <li>obs-studio-plugins.obs-vkcapture</li>
    <li>obs-studio-plugins.wlrobs</li>
    <li>olive-editor</li>
    <li>path-of-building</li>
    <li>pgmodeler</li>
    <li>pokefinder</li>
    <li>prismlauncher</li>
    <li>punes-qt6</li>
    <li>python310Packages.pyqt6</li>
    <li>python310Packages.pyqt6-charts</li>
    <li>python310Packages.pyqt6-charts.dist</li>
    <li>python310Packages.pyqt6-webengine</li>
    <li>python310Packages.pyqt6-webengine.dev</li>
    <li>python310Packages.pyqt6-webengine.dist</li>
    <li>python310Packages.pyqt6.dev</li>
    <li>python310Packages.pyqt6.dist</li>
    <li>python311Packages.pyqt6</li>
    <li>python311Packages.pyqt6-charts</li>
    <li>python311Packages.pyqt6-charts.dist</li>
    <li>python311Packages.pyqt6-webengine</li>
    <li>python311Packages.pyqt6-webengine.dev</li>
    <li>python311Packages.pyqt6-webengine.dist</li>
    <li>python311Packages.pyqt6.dev</li>
    <li>python311Packages.pyqt6.dist</li>
    <li>qalculate-qt</li>
    <li>qgnomeplatform-qt6</li>
    <li>qmmp</li>
    <li>qt6.full</li>
    <li>qt6.qmake</li>
    <li>qt6.qt3d</li>
    <li>qt6.qt3d.dev</li>
    <li>qt6.qt5compat</li>
    <li>qt6.qt5compat.dev</li>
    <li>qt6.qtbase</li>
    <li>qt6.qtbase.dev</li>
    <li>qt6.qtcharts</li>
    <li>qt6.qtcharts.dev</li>
    <li>qt6.qtconnectivity</li>
    <li>qt6.qtconnectivity.dev</li>
    <li>qt6.qtdatavis3d</li>
    <li>qt6.qtdatavis3d.dev</li>
    <li>qt6.qtdeclarative</li>
    <li>qt6.qtdeclarative.dev</li>
    <li>qt6.qtdoc</li>
    <li>qt6.qtgrpc</li>
    <li>qt6.qtgrpc.dev</li>
    <li>qt6.qthttpserver</li>
    <li>qt6.qthttpserver.dev</li>
    <li>qt6.qtimageformats</li>
    <li>qt6.qtimageformats.dev</li>
    <li>qt6.qtlanguageserver</li>
    <li>qt6.qtlanguageserver.dev</li>
    <li>qt6.qtlocation</li>
    <li>qt6.qtlocation.dev</li>
    <li>qt6.qtlottie</li>
    <li>qt6.qtlottie.dev</li>
    <li>qt6.qtmultimedia</li>
    <li>qt6.qtmultimedia.dev</li>
    <li>qt6.qtnetworkauth</li>
    <li>qt6.qtnetworkauth.dev</li>
    <li>qt6.qtpositioning</li>
    <li>qt6.qtpositioning.dev</li>
    <li>qt6.qtquick3d</li>
    <li>qt6.qtquick3d.dev</li>
    <li>qt6.qtquick3dphysics</li>
    <li>qt6.qtquick3dphysics.dev</li>
    <li>qt6.qtquickeffectmaker</li>
    <li>qt6.qtquickeffectmaker.dev</li>
    <li>qt6.qtquicktimeline</li>
    <li>qt6.qtquicktimeline.dev</li>
    <li>qt6.qtremoteobjects</li>
    <li>qt6.qtremoteobjects.dev</li>
    <li>qt6.qtscxml</li>
    <li>qt6.qtscxml.dev</li>
    <li>qt6.qtsensors</li>
    <li>qt6.qtsensors.dev</li>
    <li>qt6.qtserialbus</li>
    <li>qt6.qtserialbus.dev</li>
    <li>qt6.qtserialport</li>
    <li>qt6.qtserialport.dev</li>
    <li>qt6.qtshadertools</li>
    <li>qt6.qtshadertools.dev</li>
    <li>qt6.qtspeech</li>
    <li>qt6.qtspeech.dev</li>
    <li>qt6.qtsvg</li>
    <li>qt6.qtsvg.dev</li>
    <li>qt6.qttools</li>
    <li>qt6.qttools.dev</li>
    <li>qt6.qttranslations</li>
    <li>qt6.qttranslations.dev</li>
    <li>qt6.qtvirtualkeyboard</li>
    <li>qt6.qtvirtualkeyboard.dev</li>
    <li>qt6.qtwayland</li>
    <li>qt6.qtwayland.dev</li>
    <li>qt6.qtwebchannel</li>
    <li>qt6.qtwebchannel.dev</li>
    <li>qt6.qtwebengine</li>
    <li>qt6.qtwebengine.dev</li>
    <li>qt6.qtwebsockets</li>
    <li>qt6.qtwebsockets.dev</li>
    <li>qt6.qtwebview</li>
    <li>qt6.qtwebview.dev</li>
    <li>qt6Packages.kcoreaddons</li>
    <li>qt6Packages.kcoreaddons.bin</li>
    <li>qt6Packages.kcoreaddons.dev</li>
    <li>qt6Packages.poppler</li>
    <li>qt6Packages.poppler.dev</li>
    <li>qt6Packages.qtpbfimageplugin</li>
    <li>qt6Packages.quazip</li>
    <li>qt6Packages.quazip.dev</li>
    <li>qt6Packages.qxlsx</li>
    <li>qtcreator-qt6</li>
    <li>qtwirediff</li>
    <li>qutebrowser-qt6</li>
    <li>qutebrowser-qt6.dist</li>
    <li>retext</li>
    <li>retext.dist</li>
    <li>stellarium</li>
    <li>telegram-desktop</li>
    <li>zeal-qt6</li>
    <li>zint</li>
    <li>zint.dev</li>
  </ul>
</details>

doronbehar commented:
This release seems to have produced more `python3` references: (see also #226312):

```
/nix/store/gbi67akslfd2n5d8q4vxpy8qzih31myl-qtbase-6.5.0
└───libexec/qt-testrunner.py: …#!/nix/store/fdqpyj613dr0v1l1lrzqhzay7sk4xg87-python3-3.10.10/bin/python3.# C…
    libexec/sanitizer-testrunner.py: …#!/nix/store/fdqpyj613dr0v1l1lrzqhzay7sk4xg87-python3-3.10.10/bin/python3.# C…
    mkspecs/features/uikit/devices.py: …#!/nix/store/fdqpyj613dr0v1l1lrzqhzay7sk4xg87-python3-3.10.10/bin/python3.# C…
    → /nix/store/fdqpyj613dr0v1l1lrzqhzay7sk4xg87-python3-3.10.10
```

These scripts need to move to `$dev` output IMO.

wegank commented:
Result of `nixpkgs-review pr 224469` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>24 packages marked as broken and skipped:</summary>
  <ul>
    <li>adwaita-qt6</li>
    <li>adwaita-qt6.dev</li>
    <li>anki</li>
    <li>anki.dist</li>
    <li>anki.doc</li>
    <li>anki.man</li>
    <li>mnemosyne</li>
    <li>mnemosyne.dist</li>
    <li>olive-editor</li>
    <li>python310Packages.pyqt6-webengine</li>
    <li>python310Packages.pyqt6-webengine.dev</li>
    <li>python310Packages.pyqt6-webengine.dist</li>
    <li>python311Packages.pyqt6-webengine</li>
    <li>python311Packages.pyqt6-webengine.dev</li>
    <li>python311Packages.pyqt6-webengine.dist</li>
    <li>qt6.qtwayland</li>
    <li>qt6.qtwayland.dev</li>
    <li>qt6Packages.qtwayland</li>
    <li>qt6Packages.qtwayland.dev</li>
    <li>qutebrowser-qt6</li>
    <li>qutebrowser-qt6.dist</li>
    <li>retext</li>
    <li>retext.dist</li>
    <li>stellarium</li>
  </ul>
</details>
<details>
  <summary>100 packages built:</summary>
  <ul>
    <li>alice-tools-qt6</li>
    <li>bambootracker-qt6</li>
    <li>beamerpresenter (beamerpresenter-mupdf)</li>
    <li>beamerpresenter-poppler</li>
    <li>cutemaze</li>
    <li>dolphin-emu</li>
    <li>gpxsee-qt6</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>ladybird</li>
    <li>nanovna-saver</li>
    <li>nanovna-saver.dist</li>
    <li>pokefinder</li>
    <li>python310Packages.pyqt6</li>
    <li>python310Packages.pyqt6-charts</li>
    <li>python310Packages.pyqt6-charts.dist</li>
    <li>python310Packages.pyqt6.dev</li>
    <li>python310Packages.pyqt6.dist</li>
    <li>python311Packages.pyqt6</li>
    <li>python311Packages.pyqt6-charts</li>
    <li>python311Packages.pyqt6-charts.dist</li>
    <li>python311Packages.pyqt6.dev</li>
    <li>python311Packages.pyqt6.dist</li>
    <li>qalculate-qt</li>
    <li>qt6.qt3d (qt6Packages.qt3d)</li>
    <li>qt6.qt3d.dev (qt6Packages.qt3d.dev)</li>
    <li>qt6.qt5compat (qt6Packages.qt5compat)</li>
    <li>qt6.qt5compat.dev (qt6Packages.qt5compat.dev)</li>
    <li>qt6.qtbase (qt6Packages.qtbase)</li>
    <li>qt6.qtbase.dev (qt6Packages.qtbase.dev)</li>
    <li>qt6.qtcharts (qt6Packages.qtcharts)</li>
    <li>qt6.qtcharts.dev (qt6Packages.qtcharts.dev)</li>
    <li>qt6.qtconnectivity (qt6Packages.qtconnectivity)</li>
    <li>qt6.qtconnectivity.dev (qt6Packages.qtconnectivity.dev)</li>
    <li>qt6.qtdatavis3d (qt6Packages.qtdatavis3d)</li>
    <li>qt6.qtdatavis3d.dev (qt6Packages.qtdatavis3d.dev)</li>
    <li>qt6.qtdeclarative (qt6Packages.qtdeclarative)</li>
    <li>qt6.qtdeclarative.dev (qt6Packages.qtdeclarative.dev)</li>
    <li>qt6.qtdoc (qt6Packages.qtdoc)</li>
    <li>qt6.qtgrpc (qt6Packages.qtgrpc)</li>
    <li>qt6.qtgrpc.dev (qt6Packages.qtgrpc.dev)</li>
    <li>qt6.qthttpserver (qt6Packages.qthttpserver)</li>
    <li>qt6.qthttpserver.dev (qt6Packages.qthttpserver.dev)</li>
    <li>qt6.qtimageformats (qt6Packages.qtimageformats)</li>
    <li>qt6.qtimageformats.dev (qt6Packages.qtimageformats.dev)</li>
    <li>qt6.qtlanguageserver (qt6Packages.qtlanguageserver)</li>
    <li>qt6.qtlanguageserver.dev (qt6Packages.qtlanguageserver.dev)</li>
    <li>qt6.qtlocation (qt6Packages.qtlocation)</li>
    <li>qt6.qtlocation.dev (qt6Packages.qtlocation.dev)</li>
    <li>qt6.qtlottie (qt6Packages.qtlottie)</li>
    <li>qt6.qtlottie.dev (qt6Packages.qtlottie.dev)</li>
    <li>qt6.qtmultimedia (qt6Packages.qtmultimedia)</li>
    <li>qt6.qtmultimedia.dev (qt6Packages.qtmultimedia.dev)</li>
    <li>qt6.qtnetworkauth (qt6Packages.qtnetworkauth)</li>
    <li>qt6.qtnetworkauth.dev (qt6Packages.qtnetworkauth.dev)</li>
    <li>qt6.qtpositioning (qt6Packages.qtpositioning)</li>
    <li>qt6.qtpositioning.dev (qt6Packages.qtpositioning.dev)</li>
    <li>qt6.qtquick3d (qt6Packages.qtquick3d)</li>
    <li>qt6.qtquick3d.dev (qt6Packages.qtquick3d.dev)</li>
    <li>qt6.qtquick3dphysics (qt6Packages.qtquick3dphysics)</li>
    <li>qt6.qtquick3dphysics.dev (qt6Packages.qtquick3dphysics.dev)</li>
    <li>qt6.qtquickeffectmaker (qt6Packages.qtquickeffectmaker)</li>
    <li>qt6.qtquickeffectmaker.dev (qt6Packages.qtquickeffectmaker.dev)</li>
    <li>qt6.qtquicktimeline (qt6Packages.qtquicktimeline)</li>
    <li>qt6.qtquicktimeline.dev (qt6Packages.qtquicktimeline.dev)</li>
    <li>qt6.qtremoteobjects (qt6Packages.qtremoteobjects)</li>
    <li>qt6.qtremoteobjects.dev (qt6Packages.qtremoteobjects.dev)</li>
    <li>qt6.qtscxml (qt6Packages.qtscxml)</li>
    <li>qt6.qtscxml.dev (qt6Packages.qtscxml.dev)</li>
    <li>qt6.qtsensors (qt6Packages.qtsensors)</li>
    <li>qt6.qtsensors.dev (qt6Packages.qtsensors.dev)</li>
    <li>qt6.qtserialbus (qt6Packages.qtserialbus)</li>
    <li>qt6.qtserialbus.dev (qt6Packages.qtserialbus.dev)</li>
    <li>qt6.qtserialport (qt6Packages.qtserialport)</li>
    <li>qt6.qtserialport.dev (qt6Packages.qtserialport.dev)</li>
    <li>qt6.qtshadertools (qt6Packages.qtshadertools)</li>
    <li>qt6.qtshadertools.dev (qt6Packages.qtshadertools.dev)</li>
    <li>qt6.qtspeech (qt6Packages.qtspeech)</li>
    <li>qt6.qtspeech.dev (qt6Packages.qtspeech.dev)</li>
    <li>qt6.qtsvg (qt6Packages.qtsvg)</li>
    <li>qt6.qtsvg.dev (qt6Packages.qtsvg.dev)</li>
    <li>qt6.qttools (qt6Packages.qttools)</li>
    <li>qt6.qttools.dev (qt6Packages.qttools.dev)</li>
    <li>qt6.qttranslations (qt6Packages.qttranslations)</li>
    <li>qt6.qttranslations.dev (qt6Packages.qttranslations.dev)</li>
    <li>qt6.qtvirtualkeyboard (qt6Packages.qtvirtualkeyboard)</li>
    <li>qt6.qtvirtualkeyboard.dev (qt6Packages.qtvirtualkeyboard.dev)</li>
    <li>qt6.qtwebchannel (qt6Packages.qtwebchannel)</li>
    <li>qt6.qtwebchannel.dev (qt6Packages.qtwebchannel.dev)</li>
    <li>qt6.qtwebsockets (qt6Packages.qtwebsockets)</li>
    <li>qt6.qtwebsockets.dev (qt6Packages.qtwebsockets.dev)</li>
    <li>qt6.qtwebview (qt6Packages.qtwebview)</li>
    <li>qt6.qtwebview.dev (qt6Packages.qtwebview.dev)</li>
    <li>qt6Packages.poppler</li>
    <li>qt6Packages.poppler.dev</li>
    <li>qt6Packages.qtpbfimageplugin</li>
    <li>qt6Packages.quazip</li>
    <li>qt6Packages.quazip.dev</li>
  </ul>
</details>

NickCao committed:
qt6.qtbase: set strictDeps, remove python3 from buildInputs to reduce closure size

python3 is references in the shebang of three scripts:

/nix/store/gbi67akslfd2n5d8q4vxpy8qzih31myl-qtbase-6.5.0
└───libexec/qt-testrunner.py: …#!/nix/store/fdqpyj613dr0v1l1lrzqhzay7sk4xg87-python3-3.10.10/bin/python3.# C…
    libexec/sanitizer-testrunner.py: …#!/nix/store/fdqpyj613dr0v1l1lrzqhzay7sk4xg87-python3-3.10.10/bin/python3.# C…
    mkspecs/features/uikit/devices.py: …#!/nix/store/fdqpyj613dr0v1l1lrzqhzay7sk4xg87-python3-3.10.10/bin/python3.# C…
    → /nix/store/fdqpyj613dr0v1l1lrzqhzay7sk4xg87-python3-3.10.10

All of them are normally not used in the build process of packages thus
the shebangs batching can be safely skipped.

NickCao commented:
The dependency on python3 is now removed:
```
/nix/store/<before>-qtbase-6.5.0         648.5M
/nix/store/<after>-6.5.0                 556.4M
```
Fixes https://github.com/NixOS/nixpkgs/issues/226312

NickCao commented:
Got a hack to make texworks compile:
```
--- a/src/scripting/Script.cpp
+++ b/src/scripting/Script.cpp
@@ -350,10 +350,6 @@ Script::MethodResult Script::doCallMethod(QObject * obj, const QString& name,
                        // no return type
                        retValArg = QGenericReturnArgument();
                }
-               else if (typeName == QString::fromLatin1("QVariant")) {
-                       // QMetaType can't construct QVariant objects
-                       retValArg = Q_RETURN_ARG(QVariant, result);
-               }
                else {
                        // Note: These two lines are a hack!
                        // QGenericReturnArgument should not be constructed directly; if
```
Obviously It would break the scripting functionality though.

NickCao committed:
jami: add patch to fix annotations in bin/dbus/cx.ring.Ring.CallManager.xml

NickCao committed:
texworks: add patch to fix build with qt 6.5

NickCao committed:
mediaelch: add patch to fix build with qt 6.5

Reference: https://github.com/Komet/MediaElch/issues/1557

NickCao commented:
All regressions are now ironed out:

Result of `nixpkgs-review pr 224469` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>obs-studio-plugins.obs-ndi</li>
  </ul>
</details>
<details>
  <summary>180 packages built:</summary>
  <ul>
    <li>adwaita-qt6</li>
    <li>adwaita-qt6.dev</li>
    <li>alice-tools-qt6</li>
    <li>anki</li>
    <li>anki.dist</li>
    <li>anki.doc</li>
    <li>anki.man</li>
    <li>bambootracker-qt6</li>
    <li>beamerpresenter</li>
    <li>beamerpresenter-poppler</li>
    <li>calcmysky</li>
    <li>calibre</li>
    <li>copyq</li>
    <li>cudaPackages.nsight_compute</li>
    <li>cutemaze</li>
    <li>dolphin-emu</li>
    <li>duckstation</li>
    <li>fcitx5-chinese-addons</li>
    <li>fcitx5-configtool</li>
    <li>fcitx5-unikey</li>
    <li>fcitx5-with-addons</li>
    <li>gpxsee-qt6</li>
    <li>gqrx</li>
    <li>gqrx-gr-audio</li>
    <li>gqrx-portaudio</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>jami</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>kime</li>
    <li>ladybird</li>
    <li>libsForQt5.fcitx5-qt</li>
    <li>maestral-gui</li>
    <li>maestral-gui.dist</li>
    <li>mediaelch</li>
    <li>mediaelch-qt6</li>
    <li>mnemosyne</li>
    <li>mnemosyne.dist</li>
    <li>mozillavpn</li>
    <li>nanovna-saver</li>
    <li>nanovna-saver.dist</li>
    <li>obs-studio</li>
    <li>obs-studio-plugins.advanced-scene-switcher</li>
    <li>obs-studio-plugins.droidcam-obs</li>
    <li>obs-studio-plugins.input-overlay</li>
    <li>obs-studio-plugins.looking-glass-obs</li>
    <li>obs-studio-plugins.obs-backgroundremoval</li>
    <li>obs-studio-plugins.obs-gstreamer</li>
    <li>obs-studio-plugins.obs-hyperion</li>
    <li>obs-studio-plugins.obs-livesplit-one</li>
    <li>obs-studio-plugins.obs-move-transition</li>
    <li>obs-studio-plugins.obs-multi-rtmp</li>
    <li>obs-studio-plugins.obs-nvfbc</li>
    <li>obs-studio-plugins.obs-pipewire-audio-capture</li>
    <li>obs-studio-plugins.obs-source-record</li>
    <li>obs-studio-plugins.obs-vaapi</li>
    <li>obs-studio-plugins.obs-vkcapture</li>
    <li>obs-studio-plugins.wlrobs</li>
    <li>olive-editor</li>
    <li>path-of-building</li>
    <li>pgmodeler</li>
    <li>pokefinder</li>
    <li>prismlauncher</li>
    <li>punes-qt6</li>
    <li>python310Packages.pyqt6</li>
    <li>python310Packages.pyqt6-charts</li>
    <li>python310Packages.pyqt6-charts.dist</li>
    <li>python310Packages.pyqt6-webengine</li>
    <li>python310Packages.pyqt6-webengine.dev</li>
    <li>python310Packages.pyqt6-webengine.dist</li>
    <li>python310Packages.pyqt6.dev</li>
    <li>python310Packages.pyqt6.dist</li>
    <li>python311Packages.pyqt6</li>
    <li>python311Packages.pyqt6-charts</li>
    <li>python311Packages.pyqt6-charts.dist</li>
    <li>python311Packages.pyqt6-webengine</li>
    <li>python311Packages.pyqt6-webengine.dev</li>
    <li>python311Packages.pyqt6-webengine.dist</li>
    <li>python311Packages.pyqt6.dev</li>
    <li>python311Packages.pyqt6.dist</li>
    <li>qalculate-qt</li>
    <li>qgnomeplatform-qt6</li>
    <li>qmmp</li>
    <li>qt6.full</li>
    <li>qt6.qmake</li>
    <li>qt6.qt3d</li>
    <li>qt6.qt3d.dev</li>
    <li>qt6.qt5compat</li>
    <li>qt6.qt5compat.dev</li>
    <li>qt6.qtbase</li>
    <li>qt6.qtbase.dev</li>
    <li>qt6.qtcharts</li>
    <li>qt6.qtcharts.dev</li>
    <li>qt6.qtconnectivity</li>
    <li>qt6.qtconnectivity.dev</li>
    <li>qt6.qtdatavis3d</li>
    <li>qt6.qtdatavis3d.dev</li>
    <li>qt6.qtdeclarative</li>
    <li>qt6.qtdeclarative.dev</li>
    <li>qt6.qtdoc</li>
    <li>qt6.qtgrpc</li>
    <li>qt6.qtgrpc.dev</li>
    <li>qt6.qthttpserver</li>
    <li>qt6.qthttpserver.dev</li>
    <li>qt6.qtimageformats</li>
    <li>qt6.qtimageformats.dev</li>
    <li>qt6.qtlanguageserver</li>
    <li>qt6.qtlanguageserver.dev</li>
    <li>qt6.qtlocation</li>
    <li>qt6.qtlocation.dev</li>
    <li>qt6.qtlottie</li>
    <li>qt6.qtlottie.dev</li>
    <li>qt6.qtmultimedia</li>
    <li>qt6.qtmultimedia.dev</li>
    <li>qt6.qtnetworkauth</li>
    <li>qt6.qtnetworkauth.dev</li>
    <li>qt6.qtpositioning</li>
    <li>qt6.qtpositioning.dev</li>
    <li>qt6.qtquick3d</li>
    <li>qt6.qtquick3d.dev</li>
    <li>qt6.qtquick3dphysics</li>
    <li>qt6.qtquick3dphysics.dev</li>
    <li>qt6.qtquickeffectmaker</li>
    <li>qt6.qtquickeffectmaker.dev</li>
    <li>qt6.qtquicktimeline</li>
    <li>qt6.qtquicktimeline.dev</li>
    <li>qt6.qtremoteobjects</li>
    <li>qt6.qtremoteobjects.dev</li>
    <li>qt6.qtscxml</li>
    <li>qt6.qtscxml.dev</li>
    <li>qt6.qtsensors</li>
    <li>qt6.qtsensors.dev</li>
    <li>qt6.qtserialbus</li>
    <li>qt6.qtserialbus.dev</li>
    <li>qt6.qtserialport</li>
    <li>qt6.qtserialport.dev</li>
    <li>qt6.qtshadertools</li>
    <li>qt6.qtshadertools.dev</li>
    <li>qt6.qtspeech</li>
    <li>qt6.qtspeech.dev</li>
    <li>qt6.qtsvg</li>
    <li>qt6.qtsvg.dev</li>
    <li>qt6.qttools</li>
    <li>qt6.qttools.dev</li>
    <li>qt6.qttranslations</li>
    <li>qt6.qttranslations.dev</li>
    <li>qt6.qtvirtualkeyboard</li>
    <li>qt6.qtvirtualkeyboard.dev</li>
    <li>qt6.qtwayland</li>
    <li>qt6.qtwayland.dev</li>
    <li>qt6.qtwebchannel</li>
    <li>qt6.qtwebchannel.dev</li>
    <li>qt6.qtwebengine</li>
    <li>qt6.qtwebengine.dev</li>
    <li>qt6.qtwebsockets</li>
    <li>qt6.qtwebsockets.dev</li>
    <li>qt6.qtwebview</li>
    <li>qt6.qtwebview.dev</li>
    <li>qt6Packages.kcoreaddons</li>
    <li>qt6Packages.kcoreaddons.bin</li>
    <li>qt6Packages.kcoreaddons.dev</li>
    <li>qt6Packages.poppler</li>
    <li>qt6Packages.poppler.dev</li>
    <li>qt6Packages.qtpbfimageplugin</li>
    <li>qt6Packages.quazip</li>
    <li>qt6Packages.quazip.dev</li>
    <li>qt6Packages.qxlsx</li>
    <li>qtcreator-qt6</li>
    <li>qtwirediff</li>
    <li>qutebrowser-qt6</li>
    <li>qutebrowser-qt6.dist</li>
    <li>retext</li>
    <li>retext.dist</li>
    <li>stellarium</li>
    <li>telegram-desktop</li>
    <li>texworks</li>
    <li>zeal-qt6</li>
    <li>zint</li>
    <li>zint.dev</li>
  </ul>
</details>

nrdxp reviewed (approved):
LGTM

Anything left before we merge @NickCao?

NickCao commented:
> Anything left before we merge @NickCao?

Nothing I'm aware of.

wegank commented:
Result of `nixpkgs-review pr 224469` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>24 packages marked as broken and skipped:</summary>
  <ul>
    <li>adwaita-qt6</li>
    <li>adwaita-qt6.dev</li>
    <li>anki</li>
    <li>anki.dist</li>
    <li>anki.doc</li>
    <li>anki.man</li>
    <li>mnemosyne</li>
    <li>mnemosyne.dist</li>
    <li>olive-editor</li>
    <li>python310Packages.pyqt6-webengine</li>
    <li>python310Packages.pyqt6-webengine.dev</li>
    <li>python310Packages.pyqt6-webengine.dist</li>
    <li>python311Packages.pyqt6-webengine</li>
    <li>python311Packages.pyqt6-webengine.dev</li>
    <li>python311Packages.pyqt6-webengine.dist</li>
    <li>qt6.qtwayland</li>
    <li>qt6.qtwayland.dev</li>
    <li>qt6Packages.qtwayland</li>
    <li>qt6Packages.qtwayland.dev</li>
    <li>qutebrowser-qt6</li>
    <li>qutebrowser-qt6.dist</li>
    <li>retext</li>
    <li>retext.dist</li>
    <li>stellarium</li>
  </ul>
</details>
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>dolphin-emu</li>
  </ul>
</details>
<details>
  <summary>99 packages built:</summary>
  <ul>
    <li>alice-tools-qt6</li>
    <li>bambootracker-qt6</li>
    <li>beamerpresenter (beamerpresenter-mupdf)</li>
    <li>beamerpresenter-poppler</li>
    <li>cutemaze</li>
    <li>gpxsee-qt6</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>ladybird</li>
    <li>nanovna-saver</li>
    <li>nanovna-saver.dist</li>
    <li>pokefinder</li>
    <li>python310Packages.pyqt6</li>
    <li>python310Packages.pyqt6-charts</li>
    <li>python310Packages.pyqt6-charts.dist</li>
    <li>python310Packages.pyqt6.dev</li>
    <li>python310Packages.pyqt6.dist</li>
    <li>python311Packages.pyqt6</li>
    <li>python311Packages.pyqt6-charts</li>
    <li>python311Packages.pyqt6-charts.dist</li>
    <li>python311Packages.pyqt6.dev</li>
    <li>python311Packages.pyqt6.dist</li>
    <li>qalculate-qt</li>
    <li>qt6.qt3d (qt6Packages.qt3d)</li>
    <li>qt6.qt3d.dev (qt6Packages.qt3d.dev)</li>
    <li>qt6.qt5compat (qt6Packages.qt5compat)</li>
    <li>qt6.qt5compat.dev (qt6Packages.qt5compat.dev)</li>
    <li>qt6.qtbase (qt6Packages.qtbase)</li>
    <li>qt6.qtbase.dev (qt6Packages.qtbase.dev)</li>
    <li>qt6.qtcharts (qt6Packages.qtcharts)</li>
    <li>qt6.qtcharts.dev (qt6Packages.qtcharts.dev)</li>
    <li>qt6.qtconnectivity (qt6Packages.qtconnectivity)</li>
    <li>qt6.qtconnectivity.dev (qt6Packages.qtconnectivity.dev)</li>
    <li>qt6.qtdatavis3d (qt6Packages.qtdatavis3d)</li>
    <li>qt6.qtdatavis3d.dev (qt6Packages.qtdatavis3d.dev)</li>
    <li>qt6.qtdeclarative (qt6Packages.qtdeclarative)</li>
    <li>qt6.qtdeclarative.dev (qt6Packages.qtdeclarative.dev)</li>
    <li>qt6.qtdoc (qt6Packages.qtdoc)</li>
    <li>qt6.qtgrpc (qt6Packages.qtgrpc)</li>
    <li>qt6.qtgrpc.dev (qt6Packages.qtgrpc.dev)</li>
    <li>qt6.qthttpserver (qt6Packages.qthttpserver)</li>
    <li>qt6.qthttpserver.dev (qt6Packages.qthttpserver.dev)</li>
    <li>qt6.qtimageformats (qt6Packages.qtimageformats)</li>
    <li>qt6.qtimageformats.dev (qt6Packages.qtimageformats.dev)</li>
    <li>qt6.qtlanguageserver (qt6Packages.qtlanguageserver)</li>
    <li>qt6.qtlanguageserver.dev (qt6Packages.qtlanguageserver.dev)</li>
    <li>qt6.qtlocation (qt6Packages.qtlocation)</li>
    <li>qt6.qtlocation.dev (qt6Packages.qtlocation.dev)</li>
    <li>qt6.qtlottie (qt6Packages.qtlottie)</li>
    <li>qt6.qtlottie.dev (qt6Packages.qtlottie.dev)</li>
    <li>qt6.qtmultimedia (qt6Packages.qtmultimedia)</li>
    <li>qt6.qtmultimedia.dev (qt6Packages.qtmultimedia.dev)</li>
    <li>qt6.qtnetworkauth (qt6Packages.qtnetworkauth)</li>
    <li>qt6.qtnetworkauth.dev (qt6Packages.qtnetworkauth.dev)</li>
    <li>qt6.qtpositioning (qt6Packages.qtpositioning)</li>
    <li>qt6.qtpositioning.dev (qt6Packages.qtpositioning.dev)</li>
    <li>qt6.qtquick3d (qt6Packages.qtquick3d)</li>
    <li>qt6.qtquick3d.dev (qt6Packages.qtquick3d.dev)</li>
    <li>qt6.qtquick3dphysics (qt6Packages.qtquick3dphysics)</li>
    <li>qt6.qtquick3dphysics.dev (qt6Packages.qtquick3dphysics.dev)</li>
    <li>qt6.qtquickeffectmaker (qt6Packages.qtquickeffectmaker)</li>
    <li>qt6.qtquickeffectmaker.dev (qt6Packages.qtquickeffectmaker.dev)</li>
    <li>qt6.qtquicktimeline (qt6Packages.qtquicktimeline)</li>
    <li>qt6.qtquicktimeline.dev (qt6Packages.qtquicktimeline.dev)</li>
    <li>qt6.qtremoteobjects (qt6Packages.qtremoteobjects)</li>
    <li>qt6.qtremoteobjects.dev (qt6Packages.qtremoteobjects.dev)</li>
    <li>qt6.qtscxml (qt6Packages.qtscxml)</li>
    <li>qt6.qtscxml.dev (qt6Packages.qtscxml.dev)</li>
    <li>qt6.qtsensors (qt6Packages.qtsensors)</li>
    <li>qt6.qtsensors.dev (qt6Packages.qtsensors.dev)</li>
    <li>qt6.qtserialbus (qt6Packages.qtserialbus)</li>
    <li>qt6.qtserialbus.dev (qt6Packages.qtserialbus.dev)</li>
    <li>qt6.qtserialport (qt6Packages.qtserialport)</li>
    <li>qt6.qtserialport.dev (qt6Packages.qtserialport.dev)</li>
    <li>qt6.qtshadertools (qt6Packages.qtshadertools)</li>
    <li>qt6.qtshadertools.dev (qt6Packages.qtshadertools.dev)</li>
    <li>qt6.qtspeech (qt6Packages.qtspeech)</li>
    <li>qt6.qtspeech.dev (qt6Packages.qtspeech.dev)</li>
    <li>qt6.qtsvg (qt6Packages.qtsvg)</li>
    <li>qt6.qtsvg.dev (qt6Packages.qtsvg.dev)</li>
    <li>qt6.qttools (qt6Packages.qttools)</li>
    <li>qt6.qttools.dev (qt6Packages.qttools.dev)</li>
    <li>qt6.qttranslations (qt6Packages.qttranslations)</li>
    <li>qt6.qttranslations.dev (qt6Packages.qttranslations.dev)</li>
    <li>qt6.qtvirtualkeyboard (qt6Packages.qtvirtualkeyboard)</li>
    <li>qt6.qtvirtualkeyboard.dev (qt6Packages.qtvirtualkeyboard.dev)</li>
    <li>qt6.qtwebchannel (qt6Packages.qtwebchannel)</li>
    <li>qt6.qtwebchannel.dev (qt6Packages.qtwebchannel.dev)</li>
    <li>qt6.qtwebsockets (qt6Packages.qtwebsockets)</li>
    <li>qt6.qtwebsockets.dev (qt6Packages.qtwebsockets.dev)</li>
    <li>qt6.qtwebview (qt6Packages.qtwebview)</li>
    <li>qt6.qtwebview.dev (qt6Packages.qtwebview.dev)</li>
    <li>qt6Packages.poppler</li>
    <li>qt6Packages.poppler.dev</li>
    <li>qt6Packages.qtpbfimageplugin</li>
    <li>qt6Packages.quazip</li>
    <li>qt6Packages.quazip.dev</li>
  </ul>
</details>
